src/todo.ts

```diff
@@ -4,20 +4,19 @@
 // https://github.com/rocicorp/rails). Rails generates these CRUD functions and
 // also schema validation.
 
-import { z } from "zod";
-import { entitySchema, generate, Update } from "@rocicorp/rails";
+import { generate, Update } from "@rocicorp/rails";
 
-export const todoSchema = entitySchema.extend({
-  text: z.string(),
-  completed: z.boolean(),
-  sort: z.number(),
-});
+export type Todo = {
+  id: string;
+  text: string;
+  completed: boolean;
+  sort: number;
+};
 
-export type Todo = z.infer<typeof todoSchema>;
 export type TodoUpdate = Update<Todo>;
 export const {
   put: putTodo,
   update: updateTodo,
   delete: deleteTodo,
   list: listTodos,
-} = generate("todo", todoSchema);
+} = generate<Todo>("todo");
```

Comment:
So, this has no runtime validation. Seems a bit strange for us to set such a precence.